parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -167,7 +188,7 @@ class ModalMapDetail extends React.Component {
 const mapStateToProps = (state) => {
     return {
         error: state.parks.error,
-        events: state.events.eventsByParkId
+        events: state.events.eventsByParkId //do we need this?
```

Comment:
yeah, it's used in `this.getEventsByPark`

Reply:
for some reason whoever wrote that message made it look like it was making an api request but it's really just selecting the events from the park that's been selected through the store

Reply:
My bad, I think that was me before all the edits that ended up taking away the endpoint